dev-libs/libclangformat-ide/libclangformat-ide-9.0.1.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils llvm llvm.org toolchain-funcs
+
+DESCRIPTION="A static clangformat library patched for IDE use"
+HOMEPAGE="https://www.qt.io/blog/2019/04/17/clangformat-plugin-qt-creator-4-9"
+LLVM_COMPONENTS=( clang )
+
+llvm.org_set_globals
+
+LICENSE="Apache-2.0-with-LLVM-exceptions UoI-NCSA MIT"
+SLOT="$(ver_cut 1)"
+KEYWORDS="~amd64 ~arm ~x86"
```

Comment:
don't use `-f`. if it fails, we likely want to know.

Reply:
I assume you had meant to comment on the untested `~arm` arch.
Got it.

Reply:
err, yes. No idea how that landed there.